.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.4.6","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk-11.0.8.jdk/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/usr/local/Cellar/sbt/1.3.13/libexec/bin/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```

Comment:
Is there a new line symbol?

Reply:
I believe there shouldn't be one because this is an auto-generated file. Shouldn't this file be ignored by git, by the way?